src/bms.lisp

```diff
@@ -0,0 +1,49 @@
+(defun bms-loop () {
+    (var bms-vmin-limit-start (conf-get 'bms-vmin-limit-start))
+    (var bms-vmax-limit-start (conf-get 'bms-vmax-limit-start))
+    (var bms-tmax-limit-start (- (conf-get 'bms-t-limit-start) 3)) ; Start 3 degrees before Motor CFG -> BMS limiting functionality would happen.
+    (var bms-tmin-limit-start 0)
+    (var bms-cell-balance-start 0.5)
```

Comment:
That's what dado recommended. Cells dip especially on split packet and where there's noise.

I'm wondering if I should just remove this entirely? Or maybe only beep when the board is not running and then we could lower that threshold?

Reply:
Oh, I see. True that properly judging how the pack is balanced should only be done when idle, it can jump around a lot while riding. We could check it only when idle, question is what to do, is the board supposed to beep in that case? Is it supposed to beep continuously or once in a while? Maybe seeing the battery is imbalanced in the UI would be enough? But then, the dedicated BMS page in VESC Tool might be a better place for that...

FTR, propagating e.g. the `engaged` state to lisp and moving more of the logic over to there is exactly what I _don't_ want to do.

So this brings me around to whether we should do anything at all, and if we should do anything, seems just checking the balance is not totally out of whack, like, having a 0.5V difference, makes some sense to me :grinning: .

In that case, please add a comment along the lines of:
```
// Only check if the balance is not completely off (in idle or while riding), not meant as a proper balance check
```

Reply:
Right now I'm only checking if the board state is READY (not RUNNING) in main.c: https://github.com/lukash/refloat/pull/27/files#diff-e0cf5b28d9b6b600f0af2bc78e8fd30ec675fd731a5da86f0c4283ffc0e40176R1497

Same for BMS connection issue. Should BMS connection actually be checked while running though, and balance only alert while idle?